SConstruct

```diff
@@ -471,6 +472,8 @@ def scons():
     if args is not None:
         env.Tool('stack_analyzer', daos_prefix=build_prefix, comp_prefix=comp_prefix, args=args)
 
+    env.Alias("compiledb", env.CompilationDatabase("compile_commands.json"))
```

Comment:
Where should this file be after building?

Reply:
It should in the root directory of the project